R/tm_a_pca.R

```diff
@@ -13,16 +13,46 @@
 #' It controls the font size for plot titles, axis labels, and legends.
 #' - If vector of `length == 1` then the font sizes will have a fixed size.
 #' - while vector of `value`, `min`, and `max` allows dynamic adjustment.
-#' @templateVar ggnames "Elbow plot", "Circle plot", "Biplot", "Eigenvector plot"
-#' @template ggplot2_args_multi
+#' @param ggplot2_args `r roxygen_ggplot2_args_param("Elbow plot", "Circle plot", "Biplot", "Eigenvector plot")`
+#' @param decorators `r roxygen_decorators_param("tm_a_pca")`
```

Comment:
@m7pr moved open conversation from #812 here:

This uses an explicit template to create the parameter documentation. R inline code is the [recommended way of using templates in roxygen2](https://roxygen2.r-lib.org/articles/reuse.html?q=template#superseded)

This template is needed because of a single phrase in the `@param decorators` (see image below)

![image](https://github.com/user-attachments/assets/a23057be-2c6f-4dae-bf02-6e08196d55f1)

It's nice to have, but I'm more than happy to revert this and keep it simple as it was (ggplot2_args back to template and shared `@param decorators` with the chunk below or equivalent

```diff
# on utils.R shared_params
+ `"See section "Decorating tm_<module_name> below"`
```